test/e2e/user_workload_monitoring_test.go

```diff
@@ -141,64 +122,26 @@ func TestUserWorkloadMonitoringAlerting(t *testing.T) {
 			name: "assert alertmanager is not deployed in user namespace",
 			f:    f.AssertStatefulsetDoesNotExist("alertmanager-not-to-be-reconciled", userWorkloadTestNs),
 		},
+		// Misc
+		{
+			name: "assert containers images registry is the same than CMO",
+			f:    cmoImageRegistryIsUsedInNsAssert(t, f.UserWorkloadMonitoringNs),
+		},
+		{
+			name: "assert namespace opt out removes appropriate targets",
+			f: func(t *testing.T) {
+				assertNamespaceOptOut(t)
+			},
+		},
 	} {
-		t.Run(scenario.name, scenario.f)
-	}
-}
-
-func TestUserWorkloadMonitoringOptOut(t *testing.T) {
```

Comment:
merged them in `TestUserWorkloadMonitoring` to make use of the same setup and be able to run them in //

This check will happen on a standalone namespace `userWorkloadOptOutTestNs` so it can run in //